crates/sui-node/src/lib.rs

```diff
@@ -2086,16 +2086,16 @@ pub async fn build_http_server(
     router = router.merge(json_rpc_router);
 
     if config.enable_experimental_rest_api {
```

Comment:
i got a little confused with two different config params which seem to convey somewhat similar meanings: `enable_experimental_rest_api` and `enable_unstable_apis`. Maybe we can leave a comment somewhere about how to use them correctly

Reply:
Yeah, am planning on reworking these configs as a next step to stabilization